checks/unfixed_vulnerabilities.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checks
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	"github.com/google/go-github/v32/github"
+
+	"github.com/ossf/scorecard/checker"
+)
+
+const (
+	// UnfixedVulnerabilities is the registered name for the OSV check.
+	UnfixedVulnerabilities = "Unfixed-Vulnerabilities"
```

Comment:
Rename to `CheckUnfixedVulnerabilities` in sync with the convention of check names starting with `Check`.

Reply:
Done! Renamed to `CheckVulnerabilities` (per @inferno-chromium's suggestion to shorten the check name)